src/wagtail_personalisation/forms.py

```diff
@@ -1,19 +1,20 @@
 from datetime import datetime
+import functools
 from importlib import import_module
 
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import AnonymousUser
-from django.contrib.staticfiles.templatetags.staticfiles import static
+from django.templatetags.static import static
 from django.test.client import RequestFactory
-from django.utils.lru_cache import lru_cache
+# from django.utils.lru_cache import lru_cache
```

Comment:
Isn't it better if this line is removed?

Reply:
Yep. Was the plan. Will remove